### ISSUE TITLE
Alt+Click alternative for Viewing Thumbnail

### ISSUE THREAD

**Issue by jmedteacher** [photopea/photopea#2733]
When my students and I want to view the thumbnail of a layer mask, we are required to alt+click.
Unfortunately, we are all on chromebooks and do not have regular access to external keyboards.
Alt+Click is mapped to right click and cannot be changed for our school.

Is there an alternative to alt+click for viewing and editing a layer mask?

Thank you!

**Comment by photopea** [comment]
Hi, you can go to Chrome settings and disable this feature, to make the Alt key work normally: https://support.google.com/chromebook/thread/14888778?hl=en

I have news from Google, that they will fix it within a month or two (alt+click will not generate a right-click anymore).

**Comment by jmedteacher** [comment]
Thank you for your reply.  I attempted to find this feature in the support article, but I am unable to find anything pertaining to alt+click in the flags feature of chrome://flags I did find something about the web api overrriding keyboard shortcuts and sending directly to the web apps when in full screen mode.  I might try that if our district allows us to change it (they are very strict with what can be changed....something like a flag change would likely have to go through them.) I see nothing on alt+click as indicated in the article.  If you happen to see it, I would take a look again.
Happy to hear Google is considering fixing this.  As a Graphic Arts teacher, Photopea has been so valuable to our department!  We would love to be able to edit those masks.
Have a great day!
